index.html

```diff
@@ -2891,6 +2896,66 @@ <h3 id="applying">
         </ol>
       </section>
     </section>
+      <section>
+        <h3>
+          <code title="">iarc_rating_id</code> member
```

Comment:
Also, Also, what happens if I list this in another store? The certificate only seems to apply to one store. 

Maybe it should be:
```JSON
"iarc_certs": [
 {"id": "E84b072d-71b3-4d3e-86ae-31a8ce4e53b7", "store": "Microsoft"},
 {"id": "12dasfdf-71b3-4d3e-86ae-adsfafasfasd", "store": "Mozilla"},
]
 ```